### ISSUE TITLE
Where is the booking?

### ISSUE THREAD

**Issue by SrVaNaGaS** [kishor10d/Hotel-Room-Booking-System#2]
Hello and thanks for this good work,
I have been going through the project but I didn't find the booking part.
is this an ongoing project?

**Comment by kishor10d** [comment]
@SrVaNaGaS : Yes. This is ongoing project. It is in development phase currently. Very soon we add booking part (which is the main part) of the project. We are thinking about attractive booking UI/UX. If you have any new simple idea about UX then, please share your idea.

**Comment by truptigaonkar** [comment]
Hello Kishor,
I was going through the project and followed it. Nice project.
Can you update the booking part of this project so that we can have liittle idea how to perform hotel booking using CodeIgniter?

Thank you advance for your help.

**Comment by pravin021989** [comment]
Hey, kishor10d
 You have developed a good project but booking part is missing in this project. Please update it if completed.I am seeking for booking system, I don't have idea to developed it.I am new Lerner in CI.

**Comment by kishor10d** [comment]
@pravin021989 : Hello Pravin, I have started working on booking part. I just committed booking table in database. Also I will commit the booking code in some days.

**Comment by pravin021989** [comment]
Thank's for your reply.Please update it as soon as possible.

**Comment by kishor10d** [comment]
Its little late, but I started adding booking things in the project. Very soon I will add complete booking part with Customer and Billing part.

**Comment by bharathkumarms** [comment]
Thanks

**Comment by hamedbellal** [comment]
hey kichor10d this is really good work but the booking part is not working,
 what i have to do ?
